azure-spring-boot-samples/pom.xml

```diff
@@ -75,6 +75,11 @@
                 <artifactId>commons-io</artifactId>
                 <version>2.3</version>
             </dependency>
+            <dependency>
+                <groupId>org.projectlombok</groupId>
+                <artifactId>lombok</artifactId>
+                <scope>provided</scope>
+            </dependency>
```

Comment:
Test will still fail. Could you please move this dependency to dependencies in https://github.com/Microsoft/azure-spring-boot/blob/master/azure-spring-boot-samples/azure-active-directory-spring-boot-sample/pom.xml and remove it from this dependencyManagement? Thank you.

Reply:
Done! :-)